ui/components/component-library/badge-wrapper/README.mdx

```diff
@@ -141,8 +142,8 @@ import {
   }
 >
   <Icon
-    name={ICON_NAMES.GLOBAL}
-    size={Size.XL}
+    name={IconName.Global}
+    size={IconSize.Xl}
     color={IconColor.iconAlternative}
   />
 </BadgeWrapper>
```

Comment:
BadgeWrapper is a TS component so updating with new docs will do `component-library` folder next